backend/audit/etl.py

```diff
@@ -58,25 +58,19 @@ def load_findings(self):
         for entry in findings_uniform_guidance_entries:
             findings = entry["findings"]
             finding = Finding(
+                award_seq_number=entry.get("seq_number", 1),
```

Comment:
I believe this is the award key. There is a [PR](https://github.com/GSA-TTS/FAC/issues/1175) open to add it to FederalAward Json. The value may be a number or a string depending on ongoing discussion. It is currently a string (e.g., AWARD-#####) named [award_index](https://github.com/GSA-TTS/FAC/blob/1175-fac-bug-missing-formulas-in-federalawards-xlsx-federal-program-total-and-cluster-total-columns/backend/schemas/source/sections/FederalAwards.schema.jsonnet#L329).

Reply:
Pushed a new commit addressing this.